acceptance/analyzer_test.go

```diff
@@ -605,6 +601,35 @@ func testAnalyzer(t *testing.T, when spec.G, it spec.S) {
 					h.AssertPathExists(t, filepath.Join(copyDir, "layers", "some-buildpack-id", "some-layer.sha"))
 					h.AssertPathExists(t, filepath.Join(copyDir, "layers", "some-buildpack-id", "some-layer.toml"))
 				})
+
```

Comment:
We don't necessarily need to block this PR on it (given it might require some significant testhelper work) but it would be good to have an acceptance tests that uses anonymous auth, given that anonymous auth has been a source of bugs.

Reply:
@ekcasey @natalieparellano given that we want to cut a release tomorrow (or on Monday), should we just create an issue for that?

Reply:
I assume this is tricky b/c the registry requires auth by default?

Reply:
Hmmm... actually we have `NewDockerRegistry()` instead of `NewDockerRegistryWithAuth()` - we'll just need to deploy two registries.

Reply:
Turns out this is pretty easy to do - will add it.